style.css

```diff
@@ -593,3 +597,652 @@ form {
   border-radius: 8px;
   color: #fff;
 }
+
```

Comment:
- [ ] Please stick to the Figma design as much as possible 💯 since this project is good practice to build up your skills in HTML and CSS, currently, the about section content has some unnecessary left space that needs to get rid off 👍🏼 

### Your Design
![image](https://user-images.githubusercontent.com/68030297/229000298-16479f8b-cd19-4fbb-b505-9f6e13bc5cef.png)


### Figma Design
![image](https://user-images.githubusercontent.com/68030297/229000253-cf9c3a7c-617a-4980-97e0-90d71ae15ac8.png)

Reply:
It works appropriately According to the requirements.  Mobile version should be used in a screen 375px minimum